src/Common/src/System/Xml/Base64Encoder.cs

```diff
@@ -5,9 +5,9 @@ namespace System.Xml
 {
     internal abstract partial class Base64Encoder
     {
-        byte[] leftOverBytes;
-        int leftOverBytesCount;
-        char[] charsLine;
+        private byte[] leftOverBytes;
```

Comment:
Please do not make changes @mellinoe mentioned here. I will do it in separate commit. This is already tracked by: https://github.com/dotnet/corefx/issues/496